spec/eventually_spec.rb

```diff
@@ -9,4 +9,14 @@
     eventually { @test = outcome }
     expect(@test).to eq outcome
   end
+
+  it 'can attach to expect statements' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.